portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,122 @@
+      </h3>
+      <p>
+        During my EP internship, I implemented styling and interaction of a map displaying the geographical
+        breakdown of end users. I also worked closely with UX to implement pixel-perfect frontend components
+        according to mocks.
+      </p>
+      </section>
+
+      <section>
+      <h3>
+        <cursive>Notebook | Applicatioin Development</cursive>
+      </h3>
+      <p>
+        During my time at CSSI, I coded on a team of students to create a scrapbook application. Using Processing,
+        we implemented music, photos, and fun stickers.
+      </p>
+      </section>
+
+      <section>
+      <h3>
+        <cursive>Coffee Science | Application Design</cursive>
+      </h3>
+      <p>
+        I designed an application for a coffee shop in New Orleans. I collected data on the customers and designed
+        the application based on customer feedback.
+      </p>
+      </section>
+
+      <section>
+      <h3>
+        <cursive>Art Meets Passion | Website Design</cursive>
+      </h3>
+      <p>
+        For my senior project in high school, I partnered with InnovationV and designed a website for one
+        of their clients.
+      </p>
+      </section>
+      <hr>
+
+      <h2>
+        <cursive>Places I Would Love to Visit</cursive>
+      </h2>
+      <div class="gallery">
+        <img src="https://specials-images.forbesimg.com/imageserve/1182337590/960x0.jpg?fit=scale" alt="Bogota, Colombia" width="350" height="250">
+        <div class="desc">Bogota, Colombia</div>
+      </div>
+
+      <div class="gallery">
+        <img src="https://ceoworld.biz/wp-content/uploads/2019/11/Venice-Italy-1.jpg" alt="Venice, Italy" width="350" height="250">
+        <div class="desc">Venice, Italy</div>
+      </div>
+
+      <div class="gallery">
+        <img src="https://cdn.britannica.com/36/191736-050-5726A7F4/view-Lagos-Nigeria.jpg" alt="Lagos, Nigeria" width="350" height="250">
+        <div class="desc">Lagos, Nigeria</div>
+      </div>
+
+      <div class="gallery">
+        <img src="https://afar-production.imgix.net/uploads/syndication/holland_americas/images/Z2NcYxB3NT/original_BARCELONA_HERO__ESY-014512839.jpg?w=750&h=563&fit=crop" alt="Barcelona, Spain" width="350" height="250">
+        <div class="desc">Barcelona, Spain</div>
+      </div>
+
+      <div class="gallery">
+        <img src="https://www.earthquakeauthority.com/getattachment/a14542f8-28c3-4402-b636-d95c6fe79fd0/los-angeles-earthquake-prediction-and-risk?maxsidesize=1200&width=1200&height=500" alt="Los Angeles, California" width="350" height="250">
+        <div class="desc">Los Angeles, California</div>
+      </div>
+
+      <div class="gallery">
+        <img src="https://image-cdn.hypb.st/https%3A%2F%2Fhypebeast.com%2Fimage%2F2019%2F10%2Ftokyo-best-big-city-visit-2019-01.jpg?q=90&w=1400&cbr=1&fit=max" alt="Tokyo, Japan" width="350" height="250">
+        <div class="desc">Tokyo, Japan</div>
+      </div>
+      <br>
```

Comment:
Here's another `br` that you don't need. I'd suggest that you adjust styles instead - perhaps you just need more margin on the top of `p`s, or you could give the `p` below some special class that gives it extra space. Not a big deal to finish on a Friday for this first project, but at least something for future reference. :)